buildconfig/Jenkins/Conda/nightly_build_and_deploy.jenkinsfile

```
@@ -7,22 +7,38 @@
 // Determine default values of parameters. Some are based on the branch.
 ANACONDA_CHANNEL_DEFAULT = 'mantid'
 GITHUB_RELEASES_REPO_DEFAULT = 'mantidproject/mantid'
+// The github tag is usually only specified for release candidates. If it's left blank
+// a nightly tag is generated.
+GITHUB_RELEASES_TAG_DEFAULT = ''
 GIT_BRANCH = git_branch_name()
-// Only Publish one of the nightly builds by default. BRANCH_TO_PUBLISH is a global
+// Only Publish one of main or release-next by default. BRANCH_TO_PUBLISH is a global
 // environment variable set in the Jenkins system configuration global properties.
 PUBLISH_PACKAGES_DEFAULT = (GIT_BRANCH == "${BRANCH_TO_PUBLISH}")
+// Allow the freedom to specify GitHub and Anaconda publishing separately.
+PUBLISH_TO_ANACONDA_DEFAULT = PUBLISH_PACKAGES_DEFAULT
+PUBLISH_TO_GITHUB_DEFAULT = PUBLISH_PACKAGES_DEFAULT
+// If required, switch off the mantiddocs conda package build. This is deliberately not listed in the
+// pipeline parameters because it's only switched off for ornl-next anaconda publishing.
+BUILD_MANTIDDOCS_PACKAGE = true
 
 switch(GIT_BRANCH) {
   case ['main', 'release-next']:
     PACKAGE_SUFFIX_DEFAULT = 'Nightly'
     ANACONDA_CHANNEL_LABEL_DEFAULT = 'nightly'
-    GITHUB_RELEASES_TAG_DEFAULT = ''
     PLATFORM_CHOICES = ['all']
     break
+  case 'ornl-next':
```

This should be set on all the magic ornl fork branches
```suggestion
  case ['ornl-next', 'ornl-qa', 'ornl']:
```